### PR TITLE
Trying something new.

### PR DESCRIPTION
**Pull request by parveznukeation**

### CODE REVIEW

File: template-node.md [hunk 1]
```diff
@@ -10,7 +10,6 @@ Additional information, usage guidelines, and the like should go into the second
     - *Option B*: Description.
 
 ---
-
-[!TIP]
+<TIP>
 This is optional.
-[TIP!]
```
Comment (daxpandhi) [inline template-node.md]: @parveznukeation Why did you remove this?

Comment (parveznukeation) [reply template-node.md]: Because this is better.

Comment (daxpandhi) [reply template-node.md]: Ok.

**Comment by daxpandhi** [comment]
I don't think this adds enough value. Try a better solution and create a new pull later.
